<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_city</class>
 <widget class="QWidget" name="choose_city">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/mnjj/allmain/windowicon1.png</normaloff>:/mnjj/allmain/windowicon1.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>1200</height>
    </rect>
   </property>
   <widget class="QLabel" name="TemperatureLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="CityLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>640</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TimeLcdNumber">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>0</y>
      <width>350</width>
      <height>100</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="DateLcdNumber">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputLineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>1400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>输入您需要模拟的城市</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>640</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>当前日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>0</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>当前时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>780</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="HumLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>780</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>920</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pm25">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>920</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
